examples/example_parametrized_nodes.py

```diff
@@ -5,7 +5,7 @@
 Example demonstrating the use of Cartesian genetic programming for a
 regression task that requires fine tuning of constants in parametrized
 nodes. This is achieved by introducing a new node, "ParametrizedAdd"
-which produces a scaled and shifted version of the sum of its inputs.
+which produces a scaled and shifted version of the sum of its addresses.
```

Comment:
(see above, i would keep input here, as this is easily understandable, but what is "the sum of its addresses"?)

Reply:
(also in code below)